src/test/java/com/nerodesk/om/mock/MkDocTest.java

```diff
@@ -0,0 +1,139 @@
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.nerodesk.om.mock;
+
+import com.google.common.io.Files;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+/**
+ * Tests for {@link MkDoc}.
+ *
+ * @author Krzysztof Krason (Krzysztof.Krason@gmail.com)
+ * @version $Id$
+ */
+public final class MkDocTest {
+    /**
+     * Temporary folder.
+     * @checkstyle VisibilityModifierCheck (5 lines)
+     */
+    @Rule
+    public final transient TemporaryFolder folder = new TemporaryFolder();
+
+    /**
+     * MkDoc exists can return false for missing file.
+     * @throws IOException In case of error
+     */
+    @Test
+    public void existsReturnsFalseForMissingFile() throws IOException {
+        MatcherAssert.assertThat(
+            new MkDoc(new File(this.folder.newFolder(), "foo"), "", "")
+                .exists(),
+            Matchers.is(false)
+        );
+    }
+
+    /**
+     * MkDoc exists can return true for existing file.
+     * @throws IOException In case of error
+     */
+    @Test
+    public void existsReturnsTrueForMissingFile() throws IOException {
+        final File file = new File(this.folder.newFolder(), "bar");
+        MatcherAssert.assertThat(
+            file.createNewFile(),
+            Matchers.is(true)
+        );
+        MatcherAssert.assertThat(
+            new MkDoc(file, "", "").exists(),
+            Matchers.is(true)
+        );
+    }
+
+    /**
+     * MkDoc can delete a file.
+     * @throws IOException In case of error
+     */
+    @Test
+    public void deletesFile() throws IOException {
+        final File file = new File(this.folder.newFolder(), "test");
+        MatcherAssert.assertThat(
+            file.createNewFile(),
+            Matchers.is(true)
+        );
+        new MkDoc(file, "", "").delete();
+        MatcherAssert.assertThat(
+            file.exists(),
+            Matchers.is(false)
+        );
+    }
+
+    /**
+     * MkDoc can read from file.
+     * @throws IOException In case of error
+     */
+    @Test
+    public void readsFromFile() throws IOException {
+        final File file = new File(this.folder.newFolder(), "readable");
+        final String content = "read";
+        Files.write(content.getBytes(), file);
```

Comment:
`content.getBytes()` doesn't use encoding in this case. In method `writesToFile`, line 132, encoding is used `content.getBytes(StandardCharsets.UTF_8)`. I suppose it doesn't matter to use or no encoding for ASCII symbols, but lets use one style (everything with encoding or everithing without encoding).